panels/config/config-entries/ha-config-entries.html

```diff
@@ -13,60 +13,84 @@
-              </div>
-              <div class="card-actions">
-                <paper-button on-click='_removeEntry'>Remove</paper-button>
+                <template is='dom-repeat' items='[[_progress]]'>
+                  <paper-item>
+                    <paper-item-body two-line>
+                      [[_computeIntegrationTitle(localize, item.domain)]]
+                      <!--<div secondary>some subtitle content</div>-->
+                    </paper-item-body>
+                    <paper-button on-click='_continueFlow'>Configure</paper-button>
+                  </paper-item>
+                </template>
               </div>
             </paper-card>
-          </template>
+          </ha-config-section>
         </div>
+      </template>
 
-        <template is='dom-if' if='[[_progress.length]]'>
-          <h1>To be finished</h1>
-          <div class='entries layout horizontal wrap'>
-            <template is='dom-repeat' items='[[_progress]]'>
-              <paper-card heading='[[_computeIntegrationTitle(localize, item.domain)]]'>
-                <div class="card-actions">
-                  <paper-button on-click='_continueFlow'>Configure</paper-button>
-                </div>
-              </paper-card>
-            </template>
-          </div>
-        </template>
-
-        <h1>Add integration</h1>
-        <div class='entries layout horizontal wrap'>
-          <template is='dom-repeat' items='[[_handlers]]'>
-            <paper-card heading='[[_computeIntegrationTitle(localize, item)]]'>
-              <!-- <div class="card-content">Some content</div> -->
-              <div class="card-actions">
-                <paper-button on-click='_createFlow'>Configure</paper-button>
-              </div>
-            </paper-card>
-          </template>
-        </div>
+      <div class='content'>
+        <ha-config-section>
+          <span slot='header'>Configured</span>
+          <paper-card>
+            <div class='card-content'>
+              <template is='dom-if' if='[[!_entries.length]]'>
+                <paper-item>
+                  <paper-item-body>
+                    Nothing configured yet
+                  </paper-item-body>
+                </paper-item>
+              </template>
+              <template is='dom-repeat' items='[[_entries]]'>
+                <paper-item>
+                  <paper-item-body three-line>
+                    [[item.title]]
+                    <div secondary>Integration: [[_computeIntegrationTitle(localize, item.domain)]]</div>
+                    <div secondary>Added by: [[item.source]]</div>
+                    <div secondary>State: [[item.state]]</div>
+                  </paper-item-body>
+                  <paper-button on-click='_removeEntry'>Remove</paper-button>
+                </paper-item>
+              </template>
+            </div>
+          </paper-card>
+        </ha-config-section>
+      </div>
+
+      <div class='content'>
```

Comment:
I feel like the name `content` is no longer appropriate if used multiple times. Probably just want to add styling to `ha-config-section` ?